src/internal/m365/collection/exchange/export.go

```diff
@@ -59,19 +60,27 @@ func streamItems(
 			reader.Close()
 
 			if err != nil {
+				err = clues.Wrap(err, "reading export item")
+
+				logger.CtxErr(ctx, err).Info("processing collection item")
+
 				ch <- export.Item{
 					ID:    id,
-					Error: clues.Wrap(err, "reading data"),
+					Error: err,
```

Comment:
Will clues include the itemID?

Reply:
let me tweak the log statement a bit. Right now it will include the shortref because it's added to the context up on line 48/49

Reply:
Thanks. Line 48/49 is already outside the inner loop so I'm assuming that's the collection/directory shortref and not the item?

Reply:
arg yes, hang on, try the new version with updated clues in the context and some extra `WrapWC` calls